docs/data-sources/vuln.md

```diff
@@ -40,29 +40,29 @@ data "wallarm_vuln" "vulns" {
 
 ## Argument Reference
 
-`filter` - (Required) Filters set in the `key=value` format used to look up for vulnerability details. Possible keys:
+`filter` - (**required**) Filters set in the `key=value` format used to look up for vulnerability details. Possible keys:
 
-- `status` - (Optional) Vulnerability status. Can be: `open` for active vulnerabilities, `closed` for closed vulnerabilities, `falsepositive` for vulnerabilities marked as false positive.
-- `limit` - (Optional) Integer value defining the number of vulnerabilities to return. Possible values: 0-1000.
-- `offset` - (Optional) Integer value defining the number from which vulnerabilities should be returned. Possible values: 0 - 9199999999999999999.
+- `status` - (optional) vulnerability status. Can be: `open` for active vulnerabilities, `closed` for closed vulnerabilities, `falsepositive` for vulnerabilities marked as false positive.
+- `limit` - (optional) integer value defining the number of vulnerabilities to return. Possible values: 0-1000.
+- `offset` - (optional) integer value defining the number from which vulnerabilities should be returned. Possible values: 0 - 9199999999999999999.
```

Comment:
offset cannot be bigger than limit

Reply:
@aoleynikov-wallarm, is that right? What should be the correct "possible values"?

Reply:
Offset can be bigger than limit. Possible values for offset are between 0 and 4611686018427387903.